sentry/src/routes.rs

```diff
@@ -59,15 +59,30 @@
 //!
 //! #### GET `/v5/channel/:id/accounting`
 //!
+//! Gets all of the accounting entries for a channel from the database and checks the balances.
+//!
 //! The route is handled by [`channel::get_accounting_for_channel()`].
 //!
```

Comment:
Please keep the "Response: [`AccountingResponse`]" and add `Examples` section with the example.